upt_macports/templates/python.Portfile

```diff
@@ -18,6 +18,7 @@ distname            ${python.rootname}-${version}
 {% block versions %}
 python.versions     37
 
+## setuptools dependency is added by default and should be double checked
```

Comment:
Maybe move this comment right before line 24, so that it is close to the piece of code it refers to?